docs/concepts/alert_activations.ipynb

```diff
@@ -0,0 +1,542 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "fe59d027-121a-4411-9dc6-0c1756f205e2",
+   "metadata": {},
+   "source": [
+    "# Alert activations\n",
+    "\n",
+    "When an existing [alert](./alerts.md) is activated by its configured trigger, MLRun saves the activation record.\n",
+    "\n",
+    "## Listing alerts activations\n",
+    "This can be done in three ways:\n",
+    "\n",
+    "1. Getting activations using MLRun's Run db\n",
+    "2. Getting activations for a specific project\n",
```

Comment:
"- Getting activations using MLRun's Run db\n",
    "- Getting activations for a specific project\n",
    "- Getting activations for an alert config object"